protocol/communities/community_token_permission.go

```diff
@@ -0,0 +1,65 @@
+package communities
+
+import (
+	"reflect"
+
+	"github.com/status-im/status-go/protocol/protobuf"
+)
+
+type TokenPermissionState uint8
+
+const (
+	TokenPermissionApproved TokenPermissionState = iota
```

Comment:
Do we need a `TokenPermissionRejected` too? Maybe if an admin tries to create a TokenMaster permission or something? (they'd need to hack the client to do that, but still, we need to reject those)

Reply:
I don't think so.

> Maybe if an admin tries to create a TokenMaster permission

Then it will not go through the validation process and be silently ignored by everyone (control node, other admins and members).

One scenario where a token permission event can be explicitly rejected is when control node has newer `CommunityDescription` than the one received in events message. We have a generic solution for that already: https://github.com/status-im/status-go/pull/3876